api/src/main/resources/config/application.properties

```diff
@@ -13,7 +13,9 @@ db.showMissingCreateTableStatements=false
 spring.datasource.url=${db.composedUrl}
 
 # Elasticsearch
-es.uri=${ELASTICSEARCH_URI:localhost:9200}
+es.uri=${ELASTICSEARCH_URI:http://localhost:9200}
+es.username=${ELASTICSEARCH_USERNAME}
```

Comment:
Maybe you should comment these out after all. I don't think this auth ES work is complete yet, and I had to comment these out to start OSMT locally.